mlflow/models/dependencies_schemas.py

```diff
@@ -53,10 +56,11 @@ def set_retriever_schema(
                 other_columns=["title"],
             )
     """
-    globals()[DATABRICKS_RETRIEVER_PRIMARY_KEY] = primary_key
-    globals()[DATABRICKS_RETRIEVER_TEXT_COLUMN] = text_column
-    globals()[DATABRICKS_RETRIEVER_DOC_URI] = doc_uri
-    globals()[DATABRICKS_RETRIEVER_OTHER_COLUMNS] = other_columns or []
+    globals()[_RETRIEVER_PRIMARY_KEY] = primary_key
+    globals()[_RETRIEVER_TEXT_COLUMN] = text_column
+    globals()[_RETRIEVER_DOC_URI] = doc_uri
+    globals()[_RETRIEVER_OTHER_COLUMNS] = other_columns or []
+    globals()[_RETRIEVER_NAME] = name
```

Comment:
Can we create an object storing these attributes instead of directly inserting these attributes into `globals()`?

```python
@dataclass
class RetrieverInfo:
    primary_key: ...
    ...


info = RetrieverInfo(...)

# get
info = globals().get("__xxx_yyy___")

# delete
globals().pop("__xxx_yyy___", None)
```

Reply:
Will do this as a follow-up change